BUILD_INSTRUCTIONS.markdown

```diff
@@ -16,10 +16,16 @@ token after creating an account here:
 
 https://api.portal.trustedservices.intel.com/EPID-attestation
 
+Clone the Veracruz repository:
+
+```
+git clone git@github.com:veracruz-project/veracruz.git
+export VERACRUZ_ROOT=$PWD/veracruz
+```
 Once you have a local copy of the Veracruz source and your token:
 
 ```
-cd SGXRustOPTEEDevelDocker
+cd veracruz-docker-image
 make VERACRUZ_ROOT=<path to your veracruz directory> IAS_TOKEN=<your Intel Attestation Service token>
```

Comment:
Is the `VERACRUZ_ROOT=<path to your Veracruz directory>` part required, given the `export` that you added above?

Reply:
Good point - Will update...